dev/tools/bin/generate_gradle_lockfiles.dart

```diff
@@ -83,6 +83,7 @@ void main(List<String> arguments) {
         (loadYaml(exclusionFile.readAsStringSync()) as YamlList)
             .toList()
             .cast<String>()
+            .map((String s) => '${repoRoot.path}/$s')
```

Comment:
Exclusion was broken because of the change to how the script was intended to be run (it used to take relative directories, and was changed to absolute directories implicitly when it was changed to no longer take the output of `find ...` via a pipe operator).